docker-compose.yml

```diff
@@ -5,10 +5,10 @@ processor:
   ports:
     - "3000:3000"
   external_links:
-    - champaignakprocessor_db_1
+    - champaign_db_1
```

Comment:
@Tuuleh I'm not sure why this was changed to `champaignakprocessor_db_1`, but it broke building the container on my local machine, so I changed it backed to the linked instance. Is that OK?